invenio_app_rdm/upgrade_scripts/migrate_12_0_to_13_0.py

```diff
@@ -83,11 +83,13 @@ def run_upgrade(has, migrate):
     elif draft_success_counter > 0 or record_success_counter > 0:
         db.session.commit()
         secho(
-            f"{record_success_counter} records have been updated and {draft_error_counter} drafts have been updated",
+            f"Migration completed: {record_success_counter} records have been updated and {draft_error_counter} drafts have been updated",
```

Comment:
I mean, isn't this supposed to say `draft_success_counter` ?

Reply:
> I mean, isn't this supposed to say `draft_success_counter` ?

Oh yes, didn't notice that... just changed the message :'(